src/io/import/processors/handleConfig.ts

```diff
@@ -3,36 +3,46 @@ import { z } from 'zod';
 import { ImportHandler } from '@/src/io/import/common';
 import { useRectangleStore } from '@/src/store/tools/rectangles';
 import { useRulerStore } from '@/src/store/tools/rulers';
+import { useDataBrowserStore } from '@/src/store/data-browser';
 
-const Color = z.string();
+const color = z.string();
 
-const Label = z.object({
-  color: Color,
+const label = z.object({
+  color,
 });
 
-const RulerLabel = Label;
+const rulerLabel = label;
 
-const RectangleLabel = z.intersection(
-  Label,
+const rectangleLabel = z.intersection(
+  label,
   z.object({
-    fillColor: Color,
+    fillColor: color,
   })
 );
 
-const Config = z.object({
-  labels: z.record(Label).or(z.null()).optional(),
-  rulerLabels: z.record(RulerLabel).or(z.null()).optional(),
-  rectangleLabels: z.record(RectangleLabel).or(z.null()).optional(),
+const sampleData = z
+  .object({
+    hide: z.boolean().optional(),
```

Comment:
I would recommend making this similar in structure to the store:

```
dataBrowser: {
  hideSampleData: boolean
}
```

Reply:
Fixed.